package.json

```diff
@@ -44,9 +44,9 @@
   },
   "devDependencies": {
     "ava": "*",
-    "eslint": "^1.5.1",
+    "eslint": "^1.6.0",
     "is-plain-obj": "^1.0.0",
-    "temp-write": "^1.1.2"
+    "temp-write": "^2.0.0"
```

Comment:
You can't just bump majors and hope it works. The API changed in this major and would need to be adjusted in the code.